docs/user-guide/quickstart.md

```diff
@@ -458,6 +459,94 @@ kubectl delete svc s2 -n service-2
 kubectl delete pods s2 -n service-2
 ```
 
+<a name="hrq"/>
+
+### Hierarchical resource quotas
```

Comment:
Hi, @zfrhv
Thanks for your working!
Could you please add the point to note when using HRQ?(ref https://github.com/kubernetes-sigs/hierarchical-namespaces/issues/292)
The main point is that decimal point value cannot be specified without "" unlike ResourceQuota.

Reply:
sgtm, though please add it either at the _bottom_ of the quickstart and/or in the howto section.

Reply:
sure!
sorry idk how i missed that notification
i will add both as it sounds pretty important